core/controllers/editor_test.py

```diff
@@ -1530,6 +1532,283 @@ def test_email_functionality_cannot_be_used_by_non_moderators(self):
+                    }
+                },
+                'schema_version': 1
+            }])
+        self.playthrough_id2 = stats_models.PlaythroughModel.create(
+            self.EXP_ID, 1, 'EarlyQuit',
+            {
+                'state_name': {
+                    'value': 'state_name1'
+                },
+                'time_spent_in_exp_in_msecs': {
+                    'value': 200
+                }
+            },
+            [{
+                'action_type': 'ExplorationStart',
+                'action_customization_args': {
+                    'state_name': {
+                        'value': 'state_name2'
+                    }
+                },
+                'schema_version': 1
+            }])
+
+        stats_models.ExplorationIssuesModel.create(
+            self.EXP_ID, 1,
+            [{
+                'issue_type': 'EarlyQuit',
+                'issue_customization_args': {
+                    'state_name': {
+                        'value': 'state_name1'
+                    },
+                    'time_spent_in_exp_in_msecs': {
+                        'value': 200
+                    }
+                },
+                'playthrough_ids': [self.playthrough_id1, self.playthrough_id2],
+                'schema_version': 1,
+                'is_valid': True
+            }]
+        )
+
+        self.exp_issue_dict = {
+            'issue_type': 'EarlyQuit',
+            'issue_customization_args': {
+                'state_name': {
+                    'value': 'state_name1'
+                },
+                'time_spent_in_exp_in_msecs': {
+                    'value': 200
+                }
+            },
+            'playthrough_ids': [self.playthrough_id1, self.playthrough_id2],
+            'schema_version': 1,
+            'is_valid': True
+        }
+
+        response = self.testapp.get('/create/%s' % self.EXP_ID)
+        self.csrf_token = self.get_csrf_token_from_response(response)
+
+    def test_resolve_issue_handler(self):
+        """Test that resolving an issue deletes associated playthroughs."""
+        self.post_json(
+            '/resolveissuehandler/%s' % (self.EXP_ID),
+            {
+                'exp_issue_dict': self.exp_issue_dict,
+                'exp_version': 1
+            }, self.csrf_token)
+
+        exp_issues = stats_services.get_exp_issues(self.EXP_ID, 1)
+        self.assertEqual(exp_issues.unresolved_issues, [])
+
+        playthrough_instances = stats_models.PlaythroughModel.get_multi(
+            [self.playthrough_id1, self.playthrough_id2])
+        self.assertEqual(playthrough_instances, [None, None])
+
+    def test_error_on_passing_invalid_exp_issue_dict(self):
+        """Test that error is raised on passing invalid exploration issue
+        dict.
+        """
+        del self.exp_issue_dict['issue_type']
+        self.post_json(
+            '/resolveissuehandler/%s' % (self.EXP_ID),
+            {
```

Comment:
Please add a comment explaining why this dict is invalid. Maintainers shouldn't be forced to scroll up to `self.exp_issue_dict` to figure it out.